geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisHash.java

```diff
@@ -58,6 +59,9 @@
   private ConcurrentHashMap<UUID, List<ByteArrayWrapper>> hScanSnapShots;
   private ConcurrentHashMap<UUID, Long> hScanSnapShotCreationTimes;
   private ScheduledExecutorService HSCANSnapshotExpirationExecutor = null;
+  private static final int PER_STRING_OVERHEAD = PER_OBJECT_OVERHEAD + 46;
+  private static final int PER_HASH_OVERHEAD = PER_OBJECT_OVERHEAD + 116;
```

Comment:
I wonder if we need unit tests of these constants, so we remember to update them if someone changes the code?

Reply:
done! see RedisHashTest, RedisStringTest, and RedisSetTest